setup.py

```diff
@@ -1,29 +1,48 @@
 import os
-import glob
-import setuptools
+# from glob import glob
 from distutils.core import setup, Extension
+from Cython.Build import cythonize
+from Cython.Distutils import build_ext
 import numpy.distutils.misc_util
 
-headers = glob.glob('arrow/*.h')
-sources = glob.glob('arrow/*.c')
-obsidian = Extension('obsidian', sources=sources)
-
 with open("README.md", 'r') as readme:
 	long_description = readme.read()
 
 current_dir = os.getcwd()
 arrow_dir = os.path.join(current_dir, 'arrow')
 include = [arrow_dir] + numpy.distutils.misc_util.get_numpy_include_dirs()
 
+arrowhead = cythonize([
+	Extension('arrow.arrowhead',
+			  sources=['arrow/arrowhead.pyx'],
+			  include_dirs=['arrow'],
+			  )],
+	include_path=['arrow'],
+	# annotate=True,  # to get an HTML code listing
+	)
+
+arrow = [Extension('arrow.arrowhead',
+		   sources=['arrow/mersenne.c', 'arrow/obsidian.c', 'arrow/arrowhead.c'],
+		   include_dirs=['arrow'])]
```

Comment:
Apparently I left in two experimental variations and need to figure out which one to keep.